src/UI/Component/Dropzone/File/Wrapper.php

```diff
@@ -1,39 +1,19 @@
-<?php declare(strict_types=1);
+<?php
```

Comment:
was it intended to remove this?

Reply:
Yes I think I did, because strict_types does nothing in interfaces.